netutils/lib_mapper.py

```diff
@@ -350,6 +411,20 @@
 NETMIKO_LIB_MAPPER_REVERSE = copy.deepcopy(NETMIKO_LIB_MAPPER)
 NTCTEMPLATES_LIB_MAPPER_REVERSE = copy.deepcopy(NTCTEMPLATES_LIB_MAPPER)
 
+NETUTILS_LIBMAPPER_MAPPER = {
```

Comment:
What does this offer over "MAIN_LIB_MAPPER"?

Reply:
It appears to me that main lib mapper is more focused on the end platform, rather than the library that has a platform. I can't do `MAIN_LIB_MAPPER['aerleon']` or `MAIN_LIB_MAPPER['napalm']` unless I am mistaken?

Reply:
Any thoughts on this one?

Reply:
ok, should it be `    "netmiko": list(NETMIKO_LIB_MAPPER.keys()),` then?

Reply:
Thinking about it more, it should just be

```python
from netutils import lib_mapper

getattr(lib_mapper, f"{lib.upper()}_LIB_MAPPER")
```